node/src/types/block.rs

```diff
@@ -254,41 +254,35 @@ impl ProtoBlock {
     }
 
     /// The list of deploy hashes included in the block.
-    pub(crate) fn wasm_deploys(&self) -> &Vec<DeployHash> {
-        &self.wasm_deploys
+    pub(crate) fn deploy_hashes(&self) -> &Vec<DeployHash> {
+        &self.deploy_hashes
     }
 
     /// The list of deploy hashes included in the block.
-    pub(crate) fn transfers(&self) -> &Vec<DeployHash> {
-        &self.transfers
+    pub(crate) fn transfer_hashes(&self) -> &Vec<DeployHash> {
+        &self.transfer_hashes
     }
 
-    pub(crate) fn deploys_iter(&self) -> impl Iterator<Item = &DeployHash> {
-        self.wasm_deploys().iter().chain(self.transfers().iter())
+    pub(crate) fn deploys_transfers_iter(&self) -> impl Iterator<Item = &DeployHash> {
```

Comment:
Naming nitpick: I think this should be named `deploys_and_transfers_iter` if that's what it does, as the plurality suggested by `deploys_` and `transfers_` seems less clear as it is currently.

Reply:
Good point; done in https://github.com/CasperLabs/casper-node/pull/1266/commits/929416ca55d109a3547044d9f84532927872aff7.